Cubical/Algebra/CommRing/Instances/Int.agda

```diff
@@ -0,0 +1,26 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.CommRing.Instances.Int where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Algebra.CommRing
+open import Cubical.Data.Int
+  renaming ( ℤ to ℤType ; _+_ to _+ℤ_; _·_ to _·ℤ_; -_ to -ℤ_
+           ; +Assoc to +ℤAssoc ; +Comm to +ℤComm ; ·Assoc to ·ℤAssoc
+           ; ·Rid to ·ℤRid)
```

Comment:
Maybe easier to just do a qualified import here?

Reply:
I couldn't figure out how to get it to work for the operators, but I'll qualify the laws

Reply:
I thought you could just write things like `4 Z.+ 3`?

Reply:
I can, but the fixity of the operator doesn't seem to work unless I open the module which leads to name clashes...

Reply:
ah...